frontend/pages/policies/PolicyPage/components/PolicyQueriesErrorsListWrapper/_styles.scss

```diff
@@ -0,0 +1,140 @@
+.policies-queries-list-wrapper {
+  border-collapse: collapse;
+
+  a {
+    color: $core-vibrant-blue;
+    font-size: $x-small;
+    text-decoration: none;
+  }
```

Comment:
Future tech debt work, we should probably make this the global link styling so we can stop doing this on every page

Reply:
For sure!